### PR TITLE
End national footnote 05 014

### PR DESCRIPTION
**Pull request by alexey-klimuk**
- data migration for ending national footnote 05 014
- ignore idea files
- allow nils in relation between footnote and description
footnote_data_migration

But there is an **issue** with running data migrations multiple times locally

### CODE REVIEW

**Comment by alexey-klimuk** [comment]
[[TARIFF16] End the footnote that we added so it no longer appears](https://trello.com/c/NfCcbSWS/1538-tariff16-end-the-footnote-that-we-added-so-it-no-longer-appears)

File: app/models/footnote.rb [hunk 1]
```diff
@@ -53,7 +53,7 @@ def footnote_description
                                   right_key: [:meursing_table_plan_id, :meursing_heading_number]
 
 
-  delegate :description, :formatted_description, to: :footnote_description
+  delegate :description, :formatted_description, to: :footnote_description, allow_nil: true
```
Comment (matthewford) [inline app/models/footnote.rb]: Why is this needed, does the footnote not have a description?

Comment (alexey-klimuk) [inline app/models/footnote.rb]: Because I got a serialization error when accessing `/footnotes` page on admin

screenshot from 2017-03-14 18-16-00

with `allow_nil: true` all works

Comment (matthewford) [inline app/models/footnote.rb]: @alexey-klimuk still not sure, why would a footnote be found (in the timemachine) if its description is nil? 

Its not really valid without a description. Looking again at the migration, we should not be perhaps updating the data in the db, but adding a new record which 'ends' the old records. So adding new update records which adds the validity end date.
